test-info.php

```diff
@@ -1,3 +1,3 @@
 <?php
 
-phpinfo();
+echo phpinfo();
```

Comment:
`phpinfo()` outputs information and then returns a boolean.  I think we can leave the original file as-is.